--- DaemonStatusServiceImplService_types.py.original	2015-06-17 17:00:28.839864853 -0700
+++ DaemonStatusServiceImplService_types.py	2015-06-18 16:53:55.262067047 -0700
@@ -984,7 +984,7 @@
             def __init__(self, **kw):
                 ns = ns0.auConfigurationWsResult_Def.defParams_Dec.schema
                 TClist = [self.__class__.entry_Dec(minOccurs=0, maxOccurs="unbounded", nillable=False, encoded=kw.get("encoded"))]
-                kw["pname"] = ("http://status.ws.lockss.org/","defParams")
+                kw["pname"] = (None,"defParams")
                 kw["aname"] = "_defParams"
                 self.attribute_typecode_dict = {}
                 ZSI.TCcompound.ComplexType.__init__(self,None,TClist,inorder=0,**kw)
@@ -1033,7 +1033,7 @@
             def __init__(self, **kw):
                 ns = ns0.auConfigurationWsResult_Def.nonDefParams_Dec.schema
                 TClist = [self.__class__.entry_Dec(minOccurs=0, maxOccurs="unbounded", nillable=False, encoded=kw.get("encoded"))]
-                kw["pname"] = ("http://status.ws.lockss.org/","nonDefParams")
+                kw["pname"] = (None,"nonDefParams")
                 kw["aname"] = "_nonDefParams"
                 self.attribute_typecode_dict = {}
                 ZSI.TCcompound.ComplexType.__init__(self,None,TClist,inorder=0,**kw)
@@ -1103,7 +1103,7 @@
             def __init__(self, **kw):
                 ns = ns0.peerAgreementsWsResult_Def.agreements_Dec.schema
                 TClist = [self.__class__.entry_Dec(minOccurs=0, maxOccurs="unbounded", nillable=False, encoded=kw.get("encoded"))]
-                kw["pname"] = ("http://status.ws.lockss.org/","agreements")
+                kw["pname"] = (None,"agreements")
                 kw["aname"] = "_agreements"
                 self.attribute_typecode_dict = {}
                 ZSI.TCcompound.ComplexType.__init__(self,None,TClist,inorder=0,**kw)
@@ -1124,7 +1124,7 @@
                 def __init__(self, **kw):
                     ns = ns0.peerAgreementsWsResult_Def.agreements_Dec.entry_Dec.schema
                     TClist = [GTD("http://status.ws.lockss.org/","agreementTypeWsResult",lazy=False)(pname="key", aname="_key", minOccurs=0, maxOccurs=1, nillable=False, typed=False, encoded=kw.get("encoded")), GTD("http://status.ws.lockss.org/","peerAgreementWsResult",lazy=False)(pname="value", aname="_value", minOccurs=0, maxOccurs=1, nillable=False, typed=False, encoded=kw.get("encoded"))]
-                    kw["pname"] = ("http://status.ws.lockss.org/","entry")
+                    kw["pname"] = (None,"entry")
                     kw["aname"] = "_entry"
                     self.attribute_typecode_dict = {}
                     ZSI.TCcompound.ComplexType.__init__(self,None,TClist,inorder=0,**kw)
