--- HasherServiceImplService_client2.py	2015-08-12 14:23:44.480780323 -0700
+++ HasherServiceImplService_client.py	2015-08-07 16:12:18.222885328 -0700
@@ -84,6 +84,8 @@
 
 hashAsynchronouslyResponse = GED("http://hasher.ws.lockss.org/", "hashAsynchronouslyResponse").pyclass
 
+hash = GED("http://hasher.ws.lockss.org/", "hash").pyclass
+
 hasherParams = GTD("http://hasher.ws.lockss.org/","hasherWsParams",lazy=False)
 
 hashResponse = GED("http://hasher.ws.lockss.org/", "hashResponse").pyclass
